pkgs/os-specific/linux/usbguard/daemon_read_only_config.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/Library/ConfigFilePrivate.cpp b/src/Library/ConfigFilePrivate.cpp
+index 8aefa65..40914f7 100644
+--- a/src/Library/ConfigFilePrivate.cpp
++++ b/src/Library/ConfigFilePrivate.cpp
+@@ -51,7 +51,7 @@ namespace usbguard
+ 
+   void ConfigFilePrivate::open(const std::string& path)
+   {
+-    _stream.open(path, std::ios::in|std::ios::out);
++    _stream.open(path, std::ios::in);
```

Comment:
It does attempt to write the file, when the user tries to do save daemon settings from the applet. In order to be able to maintain the config inside the nixstore I did not see a better way. (if this tries to open rw the daemon just exists)